css/css-transforms/translate/reference/translate-and-transform-attribute-in-svg-ref.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>CSS Transforms Test: combination of the CSS translate and transform properties and the SVG transform attribute</title>
+    <link rel="author" title="Antoine Quint" href="mailto:graouts@apple.com">
+    <meta name="flags" content="svg">
+    <style type="text/css">
```

Comment:
`type="text/css` on `style` elements is redundant these days. (And in the other files.)